tfx/experimental/pipeline_testing/pipeline_recorder.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Recording pipeline from MLMD metadata."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl import app
+from absl import flags
+from collections import defaultdict
+from distutils.dir_util import copy_tree
+from ml_metadata.proto import metadata_store_pb2
+import os
+from tfx.orchestration import metadata
+
+FLAGS = flags.FLAGS
+
+default_pipeline_dir = os.path.join(os.environ['HOME'],
+                                    "tfx/pipelines/chicago_taxi_beam/")
+default_record_dir = os.path.join(os.environ['HOME'],
```

Comment:
I personally thinks that more reasonable default would be a relative path from the current directory. But not for sure in this case...